app/lib/actions/katello/host/hypervisors.rb

```diff
@@ -0,0 +1,22 @@
+module Actions
+  module Katello
+    module Host
+      class Hypervisors < Actions::EntryAction
+        def plan(environment, content_view, hypervisor_params)
+          sequence do
+            hypervisor_results = plan_action(::Actions::Candlepin::Consumer::Hypervisors, hypervisor_params)
+            return if hypervisor_results.error
```

Comment:
hah, and that came from: https://github.com/Katello/katello/blob/master/app/lib/actions/katello/system/create.rb#L39

i think we should revist that paradigm, as i assume its handling the plan phase failing (rather than the run phase as i thought it was), but for now its okay!